example-apps/rds/application.yml

```diff
@@ -1,4 +1,4 @@
-name: Exosphere-application-with-rds-dependency
+name: exosphere-application-with-rds-dependency
```

Comment:
This is the second place where we need the app name to be DNS compliant. I know we throw and error somewhere when setting up an AWS account, but should we just put this mandate somewhere in the documentation or in maybe the CLI help messages?

Reply:
We could update the error message that enforces this and add this it needs to dns compliant